.github/workflows/pullrequest.yml

```
@@ -0,0 +1,61 @@
+name: Build-PR
+on:
+  workflow_dispatch:
+    inputs:
+      PR:
+        description: PR Number
+        required: true
+  pull_request_target:
+    branches: [ master, Development ]
+    paths: Source/**
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Prebuild
+      run: |
+        git config --global user.email "actions@github.com"
+        git config --global user.name "GitHub Actions"
+        git pull --no-commit --allow-unrelated-histories -X theirs origin pull/${{ github.event.pull_request.number }}${{ github.event.inputs.PR }}/head:
+        git status
+        wget https://raw.githubusercontent.com/CombatExtended-Continued/CombatExtended/Development/Make.py -O Make.py
+        mkdir -p Assemblies
+        mkdir -p ${{ runner.temp }}/downloads
+    - name: build
+      run: |
+        TEMP=${{ runner.temp }}/ python3 Make.py --download-libs --all-libs 
+    - name: package
+      run: |
+        mkdir CombatExtended
+        cp -r Source/ Assemblies/ About/ Defs/ Languages/ Patches/ Royalty/ Sounds/ Textures/ LoadFolders.xml README.md SupportedThirdPartyMods.md CombatExtended
+        zip -9 -r build.zip CombatExtended
+    - name: Upload to DO
+      run: |
+        printf "%s" "$SSH_KEY" > id_ecdsa
+        echo "$SSH_KEY_PUB" > id_ecdsa.pub
+        chmod 600 id_ecdsa
+        echo put build.zip CombatExtended-${{ github.run_id }}.zip | sftp -oStrictHostKeyChecking=no -P 2022 -i ./id_ecdsa combatextended@combatextended.lp-programming.com
+      
+      shell: bash
+      env:
+        SSH_KEY: ${{secrets.ID_ECDSA}}
+        SSH_KEY_PUB: ${{secrets.ID_ECDSA_PUB}}
+
+    - name: Create comment
+      uses: peter-evans/create-or-update-comment@v1
+      with:
+        issue-number: ${{ github.event.pull_request.number }}
+        body: |
+          You can download the rebuilt assembly for this PR [here](https://combatextended.lp-programming.com/CombatExtended-${{ github.run_id }}.zip)
```

I'd make the link a bit more obvious

```suggestion
          You can download the rebuilt assembly for this PR here: https://combatextended.lp-programming.com/CombatExtended-${{ github.run_id }}.zip
```